front/views/AuthenticationView.tsx

```diff
@@ -32,6 +35,7 @@ const AuthenticationView = () => {
 	const [mode, setMode] = React.useState("signin" as "signin" | "signup");
 
 	return (
+		<Provider store={store}>
```

Comment:
The provider is in App.tsx, you needed a workaround ?

Reply:
for previous unit tests strategy, removed it thanks